documentation/RLC-quickstart.md

```diff
@@ -1,68 +1,73 @@
+
+:warning: Note: if you’re still generating from Swagger with RLC, please read [this doc](./RLC-Swagger-quickstart.md) for Swagger specific details.
 
-You may refer to this [link](https://github.com/Azure/azure-sdk-for-js/blob/main/CONTRIBUTING.md#prerequisites) for the environment set up prerequisites in azure-sdk-for-js repository.
 
 # Project folder and name convention
 
-Before we start, we probably should get to know the project folder and name convention for RLC libraries.
+We probably should get to know the project folder and name convention for RLC libraries.
 
-1. Project Folder structure.  
-   normally, the folder structure would be something like `sdk/{servicename}/{servicename}-{modulename}-rest`. For example, we have `sdk/agrifood/agrifood-farming-rest` folder for Farmbeats account modules. That folder will be your **${PROJECT_ROOT} folder**.  
+1. SDK Repo Root.
+   The generated libraries should be the [azure-sdk-for-js](https://github.com/Azure/azure-sdk-for-js) repo, so fork and clone it in your local then the absolute path is called **${SDK_REPO_ROOT} folder**.
+
+1. Project Folder Structure.  
+   Normally, the folder structure would be something like `sdk/{servicename}/{servicename}-{modulename}-rest`. For example, we have `sdk/agrifood/agrifood-farming-rest` folder for Farmbeats account modules under {SDK_REPO_ROOT}. That folder will be your **${PROJECT_ROOT} folder**.  
 1. Package Name Convention.  
    The package name for RLC is something like `@azure-rest/{servicename}-{modulename}`. For example, the package name for Farmbeats module is `@azure-rest/agrifood-farming`.
 
+
 # How to generate RLC
 
 We are working on to automatically generate everything right now, but currently we still need some manual work to get a releasable package. Here're the steps of how to get the package.
 
-1. **Create a swagger/README.md file.under ${PROJECT_ROOT} folder**  
-    We are using autorest to generate the code, but there's a lot of command options and in order to make the regenerate process easier in the cases of refresh the rest api input or change the code generator version, you need to document the generate command parameters.  
-    Here's an example of the swagger/README.md
+1. **Add typescript emitter dependency in package.json**  
+   
+   In Cadl project, modify `package.json` to add dependency for typescript emitter, then run `npm install` again to install `@azure-tools/cadl-typescript`.
 
-    ~~~
-    
-    # Azure  Farmbeats TypeScript Protocol Layer
-    
-    > see https://aka.ms/autorest
-    ## Configuration
+   ```json
+   "dependencies": {
+     ...
+     "@azure-tools/cadl-typescript": "1.0.0-beta.3"
```

Comment:
can we use @latest directly?

Reply:
I don't think using the @latest would be a good idea, because the reviewer would have no idea on which versions is used when generating the code.

But we add a note to update to the latest one:

```
NOTE It's always recommended to replace the version of emitter cadl-typescript with the latest version you can find in [npmjs.com](https://www.npmjs.com/package/@azure-tools/cadl-typescript) in latest tag.
```